src/test-resources/generator/yarn.lock

```diff
@@ -160,13 +160,13 @@ chownr@^1.1.1:
   resolved "https://registry.yarnpkg.com/chownr/-/chownr-1.1.4.tgz#6fc9d7b42d32a583596337666e7d08084da2cc6b"
   integrity sha512-jJ0bqzaylmJtVnNgzTeSOs8DPavpbYgEr/b0YL8/2GO3xJEhInFmhKMUnEJQjZumK7KXGFhUy89PrsJWlakBVg==
 
-chromedriver@>=92.0.0, chromedriver@^96.0.0:
-  version "92.0.1"
-  resolved "https://registry.yarnpkg.com/chromedriver/-/chromedriver-92.0.1.tgz#3e28b7e0c9fb94d693cf74d51af0c29d57f18dca"
-  integrity sha512-LptlDVCs1GgyFNVbRoHzzy948JDVzTgGiVPXjNj385qXKQP3hjAVBIgyvb/Hco0xSEW8fjwJfsm1eQRmu6t4pQ==
+chromedriver@latest:
```

Comment:
We rarely if ever use `@latest`, so this caught my attention. Here's a hypothetical scenario to consider:
- On Monday, all tests pass.
- On Tuesday, a broken (or worse, malicious) version of `chromedriver` gets released.
- On Wednesday, a PR is created. The updated version of `chromedriver` is used and fails the build in a way that is completely unrelated to the PR (or worse, does something malicious).

Would it make sense to pin the version and updated it via dependabot?

Reply:
The presence of `yarn.lock` and the use of `yarn install --frozen-lockfile` in our PR builds prevents the scenario you're describing, but I think adding an explicit entry in package.json to ensure that dependabot keeps the lockfile entry up to date is a good idea. Updated.